src/vellum/workflows/nodes/experimental/openai_chat_completion_node/node.py

```
@@ -251,6 +252,9 @@ def _create_message_content(
                 }
 
                 content.append(audio_content_item)
+            elif block.content.type == "DOCUMENT":
+                # TODO: Fill out
+                break
             else:
```

Currently, OpenAI does not support Document as a completion type:
https://platform.openai.com/docs/api-reference/chat

We can try extracting the pdf and sending it as a text source tho but not sure how much of a lift it is to do. I am mostly thinking about the work around fetching from a url and the work needed there. I think the parsing of the content should be straight forward tho. Thoughts?
cc @noanflaherty @dvargas92495

---

Yeah let's just raise a Node Exception in this case